client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -131,10 +131,13 @@ class EditUserForm extends Component {
 
 		// we may need to change the external contributor status if the user role changes
 		if ( 'roles' === event.target.name ) {
-			if ( 'administrator' !== event.target.value && true === this.state.isExternalContributor ) {
```

Comment:
Now that every role can be an external contributor reseting the status of that checkbox is confusing and unexpected behavior. I think we can safely scratch this whole function for a one-liner:
`handleChange = event => this.setState( { [ event.target.name ]: event.target.value } );`

Reply:
makes sense.  However it isn't all roles.  I will adjust so it doesn't reset if the new role selection can be external.